File: FeedleDataTier/FeedleDataTier/Network/DeleteReactionRequest.cs
using System.Text.Json.Serialization;

namespace FeedleDataTier.Network
{
    public class DeleteReactionRequest : Request
    {
        [JsonPropertyName("postReactionId")]
        public int PostReactionId { get; set; }
        public DeleteReactionRequest(int postReactionId) : base(RequestType.DeleteReactionRequest)
        {
            this.PostReactionId = postReactionId;
        }
    }
}

File: Backend/src/main/java/com/feedle/feedleapi/Networking/DeleteReactionRequest.java
package com.feedle.feedleapi.Networking;

public class DeleteReactionRequest extends Request {
    private int postReactionId;

    public DeleteReactionRequest(int postReactionId) {
        super(RequestType.DeleteReactionRequest);
        this.postReactionId = postReactionId;
    }

    public int getPostReactionId() {
        return postReactionId;
    }
}
